solidity/contracts/converter/types/standard-pool/StandardPoolConverter.sol

```diff
@@ -342,6 +344,91 @@ contract StandardPoolConverter is ConverterVersion, IConverter, ContractRegistry
         return reserveBalances(1, 2);
     }
 
+    /**
+     * @dev transfers a portion of the accumulated conversion fees
+     */
+    function processNetworkFees() external protected {
```

Comment:
why not receive a _syncReserves flag instead of 2 separate functions?

Reply:
The internal function (`processNetworkFeesAndSyncReserveBalances`) is called from a few other places, and the update of the state variable which occurs in this function, occurs at a different point in time in each one of these places (i.e., not immediately after the internal function is called).

Reply:
let's discuss this

Reply:
Renamed `processNetworkFeesAndSyncReserveBalances` to `processNetworkFees` (overloaded function).